src/repository/scriptRepository.js

```diff
@@ -6,13 +6,30 @@ const ServerError = require('../services/utils');
 const db = require('./database.js');
 const { parseTitle, parseLines, parseCharacters } = require('../services/scriptParser.js');
 
+/**
+ * @typedef Script
+ * @property {number} id Database generated script id
+ * @property {string} title Script title
+ */
 
+/** @module scriptRepo */
+
+/**
+ * get all the script titles
+ * @return {Promise<Array.<Script>>}
+ */
 async function getTitles() {
   const result = await db.query('select id, title from scripts');
 
   return result.rows.map((s) => ({ id: s.id, title: s.title }));
 }
 
+/**
+ * get the specified script
+ * @param {string} scriptDir The path for the script directory
+ * @param {string} id The script ID
+ * @return {Promise<Array<string>>}
```

Comment:
It might be nice to describe the return type here since it's not super clear _what_ the array of strings is:
```js
* @return {Promise<Array<string>>} An array of the formatted lines in the script. This is NOT the character's lines as those can be split into many formatted lines.
```